scripts/run_e2e_tests.py

```diff
@@ -37,6 +38,8 @@
 OPPIA_SERVER_PORT = 8181
 PROTRACTOR_BIN_PATH = os.path.join(
     common.NODE_MODULES_PATH, 'protractor', 'bin', 'protractor')
+PORTSERVER_SOCKET_PATH = 'portserver.socket'
+KILL_PORTSERVER_TIMEOUT = 10
```

Comment:
Include units e.g. PORTSERVER_KILL_TIMEOUT_MSECS or PORTSERVER_KILL_TIMEOUT_SECS

Reply:
done in 97bba1a74